www/klaj01/SP2/js/script.js

```diff
@@ -0,0 +1,102 @@
+"use strict";
+
```

Comment:
to jsi najednou byl osvicen nebo me prjde ze je to uplne jiny styl psani javascriptu